app/code/Magento/Newsletter/Model/Subscriber.php

```diff
@@ -95,6 +95,11 @@ class Subscriber extends \Magento\Framework\Model\AbstractModel
     protected $_customerSession;
 
     /**
+     * @var \Magento\Framework\Stdlib\DateTime\DateTime
+     */
+    protected $_date;
```

Comment:
This attribute should be `private` [Explanation](https://github.com/magento/magento2/issues/1019#issuecomment-72757989) and the name of variable should be without underscore [PSR-2#Properties](http://www.php-fig.org/psr/psr-2/#42-properties)

Reply:
Thank you, PR updated